rdt/hyper_transformer.py

```diff
@@ -346,14 +349,9 @@ def update_transformers(self, column_name_to_transformer):
         if len(self.field_transformers) == 0:
             raise Error(self._DETECT_CONFIG_MESSAGE)
 
-        invalid_columns = list(
-            set(column_name_to_transformer).difference(set(self.field_transformers)))
-        if invalid_columns:
-            raise Error(
-                f'Invalid column names: {invalid_columns}. These columns do not exist in '
-                "the config. Use 'set_config' to write and set your entire config at once."
-            )
-
+        update_columns = column_name_to_transformer.keys()
+        config_columns = self.field_transformers.keys()
```

Comment:
In both cases this is only used for the validate method so we can just get the `config_columns` inside of the method and then we don't have to pass it along